src/main/java/com/jetbulb/interviews/NotationConverter.java

```diff
@@ -0,0 +1,7 @@
+package com.jetbulb.interviews;
+
+public class NotationConverter {
+    public int toArabic(String romanNotation) throws InvalidValueException {
+        return 0;
```

Comment:
Could you please provide implementation for that or stick with just interface if it is too early for implementation?
Someone may decide that it is implemented enough already and will start to use this.